diff --git a/Source/Tools/FEXBash/FEXBash.cpp b/Source/Tools/FEXBash/FEXBash.cpp
index 00d18d319..c5283d5e9 100644
--- a/Source/Tools/FEXBash/FEXBash.cpp
+++ b/Source/Tools/FEXBash/FEXBash.cpp
@@ -47,7 +47,7 @@ int main(int argc, char** argv, char** const envp) {
   fextl::string BinShPath = RootFSPath() + "/bin/sh";
   fextl::string BinBashPath = RootFSPath() + "/bin/bash";
 
-  std::string FEXInterpreterPath = std::filesystem::path(argv[0]).parent_path().string() + "FEXInterpreter";
+  std::string FEXInterpreterPath = std::filesystem::path(argv[0]).parent_path() / "FEXInterpreter";
   // Check if a local FEXInterpreter to FEXBash exists
   // If it does then it takes priority over the installed one
   if (!std::filesystem::exists(FEXInterpreterPath)) {
diff --git a/Source/Tools/LinuxEmulation/LinuxSyscalls/Syscalls.cpp b/Source/Tools/LinuxEmulation/LinuxSyscalls/Syscalls.cpp
index bcb92096a..35ae4196b 100644
--- a/Source/Tools/LinuxEmulation/LinuxSyscalls/Syscalls.cpp
+++ b/Source/Tools/LinuxEmulation/LinuxSyscalls/Syscalls.cpp
@@ -164,12 +164,6 @@ static bool IsShebangFile(std::span<char> Data) {
     // Executable argument
     fextl::string& ShebangProgram = ShebangArguments[0];
 
-    // If the filename is absolute then prepend the rootfs
-    // If it is relative then don't append the rootfs
-    if (ShebangProgram[0] == '/') {
-      ShebangProgram = FEX::HLE::_SyscallHandler->RootFSPath() + ShebangProgram;
-    }
-
     return FHU::Filesystem::Exists(ShebangProgram);
   }
 
